### DISCUSSION TITLE
I found few bugs on github

### DISCUSSION THREAD

**Discussion by itsdotnet** [community/community#68963] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

After that I submit about that to **Bug Bounty Program**  and my first report is closed 
Then I submit second report now I thinks its on review but I sebmitted it few days ago.

My point is getting this badge who can hepl me

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @itsdotnet ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
